monkey/infection_monkey/monkey.py

```diff
@@ -232,6 +233,8 @@ def start(self):
                     if not self._keep_running:
                         break
 
+                start_ransomware()
+
```

Comment:
No if statement? We should check the config before calling this.

Reply:
No if statement. `monkey.py` should express the order of execution, not logic.